doc/api/fs.md

```diff
@@ -2938,7 +2937,6 @@ The following constants are meant for use with the [`fs.Stats`][] object's
 [`util.inspect(stats)`]: util.html#util_util_inspect_object_options
 [`util.promisify()`]: util.html#util_util_promisify_original
 [Caveats]: #fs_caveats
-[Common System Errors]: errors.html#errors_common_system_errors
```

Comment:
Oops, it was used indeed. I just updated it and re-check them all manually. Thanks!